target/linux/ipq40xx/image/generic.mk

```diff
@@ -196,6 +200,32 @@ define Device/asus_map-ac2200
 endef
 TARGET_DEVICES += asus_map-ac2200
 
+# WARNING: this is an initramfs image that gets you half of the way there
+#          you need to delete the jffs2 ubi volume and sysupgrade to the final image
+# to get a "trx" you can flash via web UI for ac42u/ac58u:
+# - change call Device/FitImageLzma to Device/FitImage
+# - add the following below UIMAGE_NAME
+#   UIMAGE_MAGIC := 0x27051956
+#   IMAGES += factory.trx
+#   IMAGE/factory.trx := copy-file $(KDIR)/tmp/$$(KERNEL_INITRAMFS_IMAGE) | uImage none
+define Device/asus_rt-ac42u
+	$(call Device/FitImageLzma)
+	DEVICE_VENDOR := ASUS
+	DEVICE_MODEL := RT-AC42U
+	SOC := qcom-ipq4019
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DTB_SIZE := 65536
+	IMAGE_SIZE := 20439364
```

Comment:
Do we need this?
It seems `DTB_SIZE` was copied from apm821xx, but none of ipq40xx build recipe does not utilize it.
`IMAGE_SIZE` value seems weird. I would just drop it as it is not used by the image recipe anyway.

Reply:
@chunkeey I found you're using the `IMAGE_SIZE` value in your latest staging tree, would you mind if I remove it?

Reply:
I'll keep `IMAGE_SIZE`. I don't know if it's applicable on the artifact image (by `check-size`) but at least it doesn't hurt.
But I'll remove `DTB_SIZE` as it is not added to `DEVICE_VARS` in ipq40xx. It can be re-added later with a proper support.